moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -459,11 +459,11 @@ const ProjectFundingTable = ({ handleSnackbar }) => {
         : updateProjectFundingData.funding_description;
 
     updateProjectFundingData.funding_source_id =
-      updateProjectFundingData.moped_fund_source?.funding_source_id;
+      updateProjectFundingData.moped_fund_source?.funding_source_id || null;
```

Comment:
the optional chaining was working but the value was falling back to `undefined` which was being compared for equality against `null` which seemed like an edit.

Reply:
pesky `undefined`'s!